README.md

```diff
@@ -209,321 +208,7 @@ class Node extends Libp2p {
-
-##### `libp2p.stats.forTransport(transportTag).movingAverages`
-
-Returns an object containing the following keys:
-
- dataSent
- dataReceived
-
-Each one of them contains an object that has a key for each interval (`60000`, `300000` and `900000` milliseconds).
-
-Each one of these values is [an exponential moving-average instance](https://github.com/pgte/moving-average#readme).
-
-#### Per-protocol stats
-
-##### `libp2p.stats.protocols()`
-
-Returns an array containing the tags (string) for each observed protocol.
-
-##### `libp2p.stats.forProtocol(protocolTag).snapshot`
-
-Should return a stats snapshot, which is an object containing the following keys and respective values:
-
-- dataSent: amount of bytes sent, [Big](https://github.com/MikeMcl/big.js#readme) number
-- dataReceived: amount of bytes received, [Big](https://github.com/MikeMcl/big.js#readme) number
-
-
-##### `libp2p.stats.forProtocol(protocolTag).movingAverages`
-
-Returns an object containing the following keys:
-
-- dataSent
-- dataReceived
-
-Each one of them contains an object that has a key for each interval (`60000`, `300000` and `900000` milliseconds).
-
-Each one of these values is [an exponential moving-average instance](https://github.com/pgte/moving-average#readme).
-
-#### Per-peer stats
-
-##### `libp2p.stats.peers()`
-
-Returns an array containing the peerIDs (B58-encoded string) for each observed peer.
-
-##### `libp2p.stats.forPeer(peerId:String).snapshot`
-
-Should return a stats snapshot, which is an object containing the following keys and respective values:
-
-- dataSent: amount of bytes sent, [Big](https://github.com/MikeMcl/big.js#readme) number
-- dataReceived: amount of bytes received, [Big](https://github.com/MikeMcl/big.js#readme) number
-
-
-##### `libp2p.stats.forPeer(peerId:String).movingAverages`
-
-Returns an object containing the following keys:
-
-- dataSent
-- dataReceived
-
-Each one of them contains an object that has a key for each interval (`60000`, `300000` and `900000` milliseconds).
-
-Each one of these values is [an exponential moving-average instance](https://github.com/pgte/moving-average#readme).
-
-#### Stats update interval
-
-Stats are not updated in real-time. Instead, measurements are buffered and stats are updated at an interval. The maximum interval can be defined through the `Switch` constructor option `stats.computeThrottleTimeout`, defined in milliseconds.
-
-### Private Networks
-
-#### Enforcement
-
-Libp2p provides support for connection protection, such as for private networks. You can enforce network protection by setting the environment variable `LIBP2P_FORCE_PNET=1`. When this variable is on, if no protector is set via `options.connProtector`, Libp2p will throw an error upon creation.
-
-#### Protectors
-
-Some available network protectors:
-* [libp2p-pnet](https://github.com/libp2p/js-libp2p/tree/master/src/pnet)
+See [API.md](./doc/API.md).
```

Comment:
I will add in the section before the configuration doc, which will also be linked from the API when relevant